pkg/mover/tar.go

```diff
@@ -65,3 +69,93 @@ func (tfw *tarFileWriter) WriteIOFile(name string, size int64, r io.Reader, perm
 	}
 	return nil
 }
+
+func untar(tarFile, src, dst string) error {
```

Comment:
fileName?

Reply:
I have documented this and the other tar function added. Also renamed the arguments.